applications/nrf_desktop/src/events/hid_event.c

```diff
@@ -17,8 +17,9 @@ static int log_hid_report_event(const struct event_header *eh, char *buf,
 
 	__ASSERT_NO_MSG(event->dyndata.size > 0);
 
-	pos = snprintf(buf, buf_len, "Report 0x%x send to %p:",
+	pos = snprintf(buf, buf_len, "Report 0x%x src:%p sub:%p: ",
```

Comment:
the last space in format text is not needed

Reply:
done